source/scpp/src/crypto/SHA.cpp

```diff
@@ -10,76 +10,76 @@
 namespace stellar
 {
 
-// Plain SHA256
-uint256
-sha256(ByteSlice const& bin)
+// Plain SHA512
```

Comment:
So I guess you want to do the migration in two steps ?

Reply:
I just realized this could have been all part of the same PR. But I'll leave it as is then, and fix up the comments.